src/main/resources/db/changelog/008-add-constraints-on-columns-in-database-tables.yaml

```diff
@@ -0,0 +1,8 @@
+databaseChangeLog:
+  - changeSet:
+      id: "10"
+      author: zetxdev
+      changes:
+        - sqlFile:
+            dbms: 'mysql'
+            path: db/changelog/constrains.sql
```

Comment:
Please reneme name of thisfile to:
`008-add-constraints-on-columns-in-database-tables.sql`